package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webpack-setup",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "webpack --mode production",
+    "dev": "webpack serve"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ulisesedu7/Webpack-Setup.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ulisesedu7/Webpack-Setup/issues"
+  },
+  "homepage": "https://github.com/ulisesedu7/Webpack-Setup#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.72.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.9.1"
+  }
+}
```

Comment:
- You seem to have installed webpack wrongly, I have some errors starting your project on my machine 👇 👇 . I suggest you add this `"start": "webpack serve --open",` to your scripts in the package.json file. Please try to fix this

Error:
<img width="440" alt="webpack" src="https://user-images.githubusercontent.com/73607512/194716075-212e38d3-fe28-4c39-a435-0cd3a5af64e8.PNG">